src/app/wallet-activation/user-activation/user-activation.component.ts

```diff
@@ -16,18 +16,22 @@ export class UserActivationComponent implements OnInit {
 
     users: UserActivationModel[];
 
-    constructor(private activationService: WalletActivationService, private broadService: BroadcastService) {
+    constructor(private activationService: WalletActivationService,
+                private broadService: BroadcastService) {
     }
 
     ngOnInit() {
-        SpinnerUtil.showSpinner();
+        this.fetchWalletToActivate();
+    }
 
-        this.activationService.getUnactivatedWallets('user').subscribe((res: any) => {
+    fetchWalletToActivate() {
+        SpinnerUtil.showSpinner();
+        this.activationService
+            .getUnactivatedWallets('user').subscribe((res: any) => {
```

Comment:
Merge recently updated master branch. 
From now on, every result that comes from the service should be strongly typed with the interface. With that, we should avoid casting parameters as `any`.

So, wherever we use:
`.subscribe((res: any) => {})`,
we should change it to 
`.subscribe(res => {})`.

Reply:
@mdrusko1 , maybe you forgot, can you apply changes for this PR?